### PR TITLE
feat(clean): tuple of input formats for clean_country()

### LINKED ISSUE #567: clean: validate_country() cannot work as expected
**Describe the bug**
when I test validate_country, I found it treated string number as country names: (df['str'] is string type)
image

**Expected behavior**
It should return "false false false".

### PR DESCRIPTION
**Pull request by ryanwdale**
# Description

Adds support for cleaning multiple input formats at once by passing a tuple of input formats to the clean_country() and validate_country() functions.

closes #567

# How Has This Been Tested?

Unit tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have already squashed the commits and make the commit message conform to the project standard.
- [ ] I have already marked the commit with "BREAKING CHANGE" or "Fixes #" if needed.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: dataprep/clean/clean_country.py [hunk 1]
```diff
@@ -344,3 +355,46 @@ def _get_format_from_name(name: str) -> str:
...
+    """
+    Converts a string input format to a tuple of allowed input formats and raises an error
+    if an input format is not valid.
+    """
+    input_formats = {"auto", "name", "official", "alpha-2", "alpha-3", "numeric"}
+    if isinstance(input_format, str):
+        if input_format == "auto":
+            input_format = ("name", "official", "alpha-2", "alpha-3", "numeric")
+        else:
+            input_format = (input_format,)
```
Comment (brandonlockhart) [inline dataprep/clean/clean_country.py:392]: What do you think about
``` python
if input_format == "auto":
    return ("name", "official", "alpha-2", "alpha-3", "numeric")
input_format = (input_format,)
```

**Review by brandonlockhart — CHANGES_REQUESTED** [review]
Looks good @ryanwdale. I just left a small comment, then we can merge.

File: dataprep/clean/clean_country.py (hunk 1 above)
Comment (ryanwdale) [reply dataprep/clean/clean_country.py]: That looks a lot better thanks

**Comment by ryanwdale** [comment]
Thanks for the review @brandonlockhart, I made the change

**Review by brandonlockhart — APPROVED** [review]

**Pull request merged**